### DISCUSSION TITLE
Run torchmd without prior

### DISCUSSION THREAD

**Discussion by AntonioMirarchi** [torchmd/torchmd#27] [Q&A]
Hi, I'm trying to run torchmd loading a ckpt file from torchmdnet without prior. There is a problem when the code try to compute the LJ-term on line 106 of the parameters.py. We are using forceterms = [] in the conf file. So how can I run a MD loading all the parameters from the torchmd-net cktp without prior?

**Comment by MaciejMajew** [comment]
what is the error?

**Comment by MaciejMajew** [comment]
It's probably trying to precompute the LJ params. They won't be used in the sim if you don't include any forceterms. As a quick fix you can include dummy values in the forcefield file. A proper solution needs to be implemented by @stefdoerr. But I'm not sure if it's worth the effort since we are aiming to run NNPs with openMM plugin (it's faster).

**Comment by giadefa** [comment]
it's an if then, so it's probably worth it.

The if then is there for bonds, etc but not for LJ
